lib/marc_cleanup/generate_field_yaml.rb

```diff
@@ -0,0 +1,67 @@
+### Derived from
+###   https://github.com/reeset/marcrulesfiles/blob/master/marcrules.txt;
+### 1xx and require 245 rules were stripped, in addition to the stub fields:
```

Comment:
Does this mean these fields were stripped from a local copy of marcrules.txt?

Reply:
Yes, I downloaded the file and made some changes to it. Should I document all the changes I had to make to the file?

Reply:
Maybe just commit the before and after versions of marcrules.txt?

Reply:
I don't really want to have an extra text file in the repo. My hope was that my narrative of all the steps I took to modify the file would be enough to 'show my work'. Should I just get rid of the generate_field_yaml file altogether, since it's not really functionality of the marc_cleanup repo? I deleted the file that I used to generate the full yaml, and it would take me a day to reconstruct it.

Reply:
Ok. Thinking in terms of reproducibility, I think it's good to keep generate_field_yaml, or at least link to it as a Gist.

Reply:
So are the comments in the code enough? With the comments as they stand and the YAML file that is in the codebase, would you be able to reconstruct how to do it?